### PR TITLE
Refactor: Import socket variable locally for logging events

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: Import socket variable locally for logging events
Refactor: Bump version to 1.3.9

### CODE REVIEW

**Comment by praves77** [comment]
Does above PR also fix the `2022/05/16 18:11:29 [error] 8#8: *32 [lua] connection.lua:26: get_connection(): [moesif] failed to connect to api.moesif.net:443: [api.moesif.net](http://api.moesif.net/) could not be resolved (110: Operation timed out), context: ngx.timer, client: 172.20.0.1, server: 0.0.0.0:80 issue?` I could not understand how import would fix connection issue unless the [code](https://github.com/Moesif/lua-resty-moesif/blob/4694f056715921b34e2d2a08332ce46f36aa3bad/lua/resty/moesif/log.lua#L318) was using un-imported lib?

**Comment by praves77** [comment]
Other changes look fine.

File: lua/resty/moesif/send_event.lua [hunk 1]
```diff
@@ -78,6 +78,9 @@ end
 
 if isempty(config:get("batch_max_time")) then
   config:set("batch_max_time", 2)
+elseif config:get("batch_max_time") > 30 then 
+  ngx.log(ngx.ERR, "[moesif] Batch max time config value (" .. tostring(config:get("batch_max_time")) .. ") should be less than or equal to 30 seconds");
```
Comment (praves77) [inline lua/resty/moesif/send_event.lua:82]: We should change the message about resetting it to max ceiling value, like `"[moesif] Resetting Batch max time config value (" .. tostring(config:get("batch_max_time")) .. ") to max allowed (30 seconds)"`

**Review by dgilling — APPROVED** [review]

**Comment by praves77** [comment]
It will be good to update the message otherwise it looks fine.

**Review by praves77 — APPROVED** [review]

File: lua/resty/moesif/send_event.lua (hunk 1 above)
Comment (keyur9) [reply lua/resty/moesif/send_event.lua]: have updated the log message

**Pull request merged**